CMakeLists.txt

```diff
@@ -83,6 +83,7 @@ if(NOT WIN32)
 endif()
 if(CORE_SYSTEM_NAME STREQUAL linux)
   option(ENABLE_EVENTCLIENTS    "Enable event clients support?" OFF)
+  option(ADDONS_CONFIGURE_AT_STARTUP "Configure binary addons at startup?" ON)
```

Comment:
alright, i missed the default in the first commit, thus the question how le and fp would activate :)